plugins/file_logs.yaml

```
@@ -1,4 +1,4 @@
-version: 0.1.0
+version: 0.1.1
```

```suggestion
version: 0.2.0
```
We've been bumping the minor version of these plugins for these changes, since it adds functionality.